client/src/browser-table.html

```diff
@@ -289,7 +289,7 @@ <h1 label>Browser support</h1>
       </div>
       <div class="row">
         <a spec href="https://w3c.github.io/webcomponents/spec/imports/" target="_blank" rel="noopener">
-          <div label="HTML imports"><svg viewBox="0 0 26 22"><use xlink:href="/sprite.octicons.svg#import"></use></svg></div>
+          <div label="HTML Imports"><svg viewBox="0 0 26 22"><use xlink:href="/sprite.octicons.svg#import"></use></svg></div>
```

Comment:
I wonder if we should simply remove this HTML imports from this table. WDYT?

Reply:
No-no-no, we in no case should not delete HTML Imports. Almost all existing web components (at least published on https://webcomponents.org) still use HTML Imports. ES modules support lands in all browsers and Polymer 3.0 released just yesterday. :)

Reply:
@ExE-Boss Could you please provide some proof that HTML Imports spec has been actually deprecated? :-/ I know, that Firefox and Safari teams has no plans to implement HTML Imports support, while the current status of HTML Imports in Edge is ["Under consideration"](https://developer.microsoft.com/en-us/microsoft-edge/platform/status/htmlimports/). But I have not yet heard that it's officially deprecated. What about "Not recomended"? ;-)

Reply:
Yeah, <q>Not recommended</q> is probably better.